### ISSUE TITLE
Get KubeCon listed in calendar rich text formats for Kubernetes

### ISSUE THREAD

**Issue by dankohn** [LF-Engineering/lfevents#468]
In an incognito window, Google for `[Kubernetes conference]`. https://www.google.com/search?q=kubernetes+conference

![image](https://user-images.githubusercontent.com/3083270/74048594-86992580-49a0-11ea-808c-45d593948ad7.png)

KubeCon isn't coming up here, but two other events are, even though they don't have Kubernetes in their name either. Please look at the metadata of QCon and DevOpsCon and figure out how we can add similar metadata to show up in the structured events results of that search page.

**Comment by cjyabraham** [comment]
I did a bit of playing around with this and discovered the following:
- the kubecon events are listed when you do a more specific search, such as for "kubecon 2020 amsterdam", so this is good
- if you search for just "kubernetes conference" it looks like the results are very localized to where google thinks you are.  I moved around on my VPN a few times.  I searched from Canada and got no events listed.  I searched from an IP address in LA and got some LA events.  I search from an IP in London and got a bunch of London events.   I notice the ones you got are all in NYC.

This makes me think the search is hyper-localized and behaving as it should and there likely isn't a way to game it to show an Amsterdam event for a broad search by a user in NYC.
